apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -328,7 +319,6 @@ const styles = {
   button: {
```

Comment:
i wonder if the styles "just work" if we move them all to the existing CSS module?

Reply:
Good call, I'll pull these into CSS